lighthouse-core/report/html/renderer/dom.js

```diff
@@ -77,6 +77,25 @@ class DOM {
     return element;
   }
 
+  /**
+   * @param {Document} document
+   */
+  removeCommentNodes(document) {
+    const walker = document.createTreeWalker(document, 128, null);
+
+    const commentNodes = [];
+    let currentNode;
+    while (currentNode = walker.nextNode()) {
+      commentNodes.push(currentNode);
+    }
+
+    for (const node of commentNodes) {
```

Comment:
for (const node of commentNodes) node.remove();

Reply:
ok. had to convince tsc.